midend/parserUnroll.cpp

```diff
@@ -280,7 +282,7 @@ class ParserSymbolicInterpreter {
             if (value == nullptr)
                 value = factory->create(type, true);
             if (value->is<SymbolicError>()) {
-                ::error(ErrorType::ERR_EXPRESSION,
+                ::warning(ErrorType::ERR_EXPRESSION,
```

Comment:
is there a way to distinguish between possible errors and certain errors? We want to give warnings for the former and errors for the latter. ("may" and "must" in compiler terminology.)

Reply:
There are some comments on SymbolicError and SymbolicStaticError which suggest that this is indeed true.
But the comments on SymbolicError seem wrong.

Reply:
I think that ParserUnroll should generate some message which helps user to understand the reason for unsuccessful unrolling. In this case a program should continue execution, because it is just a problem of interpretation inside parserUnroll.